### ISSUE TITLE
Support For Client Credentials Flow

### ISSUE THREAD

**Issue by ssinno28** [virencehealth/apiserver.virencehealth.com#37]
I'm trying to integrate with the Centricity API using Client Credentials Flow, but every example I've seen it seems like we have to have the patient log in before we can make any API calls. Is it possible to just do server to server authentication or do we need the user to sign in.

For example can I get a token strickly from our server api and then schedule appointments for a patient from there or does the patient always have to log in for us to get a token?

**Comment by SameerSingh1991** [comment]
possible duplicate of #26

**Comment by SameerSingh1991** [comment]
Closing this issue because it has been inactive for a few months. Please reopen if you still encounter this issue.
